search/src/dockit-search.ts

```diff
@@ -0,0 +1,260 @@
+        /\0(.*?)\0/g,
+        '<strong class="highlight">$1</strong>'
+      );
+    }
+  }
+
+  hideHits() {
+    this.searchHits.style.display = 'none';
+    this.searchOverlay.style.display = 'none';
+    this.searchInput.value = '';
+    this.searchInput.placeholder = this.placeHolder;
+  }
+
+  get placeHolder() {
+    return window.navigator.platform === 'MacIntel' ? '⌘K' : 'Ctrl+K';
+  }
+
+  get searchInput(): HTMLInputElement {
+    return this.shadowRoot.querySelector('#search-input');
+  }
+
+  get searchHits(): HTMLUListElement {
+    return this.shadowRoot.querySelector('#search-hits');
+  }
+
+  get searchOverlay(): HTMLDivElement {
+    return this.shadowRoot.querySelector('#search-overlay');
+  }
+
+  static get styles() {
+    return css`
+      #search-box {
+        position: relative;
+        flex-grow: 1;
+        margin-left: 1rem;
+      }
+      #search-box form {
+        z-index: 11;
+        display: flex;
+        align-items: center;
+      }
+      #search-box form label {
+        position: relative;
+        width: 100%;
+      }
+      #search-box form label:before {
+        content: '';
+        z-index: 12;
+        position: absolute;
+        left: 10px;
+        top: 0;
+        bottom: 0;
+        width: 20px;
+        background: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='32' height='32' viewBox='0 0 20 20' fill-rule='evenodd'%3E%3Cpath d='M14.386 14.386l4.0877 4.0877-4.0877-4.0877c-2.9418 2.9419-7.7115 2.9419-10.6533 0-2.9419-2.9418-2.9419-7.7115 0-10.6533 2.9418-2.9419 7.7115-2.9419 10.6533 0 2.9419 2.9418 2.9419 7.7115 0 10.6533z' stroke='gray' fill='none' fill-rule='evenodd' stroke-linecap='round' stroke-linejoin='round' %3E%3C/path%3E%3C/svg%3E")
+          center / contain no-repeat;
+      }
+      #search-overlay {
+        display: none;
+        position: fixed;
+        z-index: 10;
+        background-color: #1f1f1f0f;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+      }
+      #search-label {
+        padding: 0.5rem;
+        display: flex;
+      }
+      #search-input {
+        border: 1px solid var(--dockit-layout-header-border-color);
+        border-radius: 0.5rem;
+        padding: 0.5rem;
+        padding-left: 2.5rem;
+        width: 100%;
+        z-index: 11;
+        position: relative;
+        background: var(--dockit-layout-bg);
+        font-size: large;
+        color: inherit;
+      }
+      #search-input:focus {
+        outline: 1px solid var(--dockit-layout-accent);
+      }
+      #search-hits {
+        display: none;
+        position: absolute;
+        right: 0;
+        left: 0;
+        list-style: none;
+        overflow: auto;
+        max-height: 70vh;
+        border: 1px solid var(--dockit-layout-header-border-color);
+        background: var(--dockit-layout-bg);
+        z-index: 11;
+        border-radius: 0.5rem;
+        padding: 0.5rem;
+        @media only screen and (max-width: 768px) {
```

Comment:
`max-width: 768px` looks to be a new value for `breakpoints.js` tokens

Reply:
good point

Reply:
done